CHANGELOG.md

```diff
@@ -35,14 +35,14 @@ end
 **Added**:
 
 - **decidim-proposals**: Add support to import .odt participatory text files. [\#4386](https://github.com/decidim/decidim/pull/4386)
+- **decidim-proposals**: Admin can create proposals from the admin panel, with a meeting as an author.[\#4382](https://github.com/decidim/decidim/pull/4382)
```

Comment:
Now your changelog entry is missing 😅

Reply:
No, it's present. But in second line :confused:

Reply:
Fixed!